src/sql-parser/src/parser.rs

```diff
@@ -1649,7 +1649,14 @@ impl<'a> Parser<'a> {
             };
             Format::Csv { columns, delimiter }
         } else if self.parse_keyword(JSON) {
-            Format::Json
+            if self.parse_keywords(&[USING, SCHEMA]) {
```

Comment:
nit: I think we'd wanted to call this `USING TYPE` to leave `USING SCHEMA` for a possible future JSON schema-based decoding?